### ISSUE TITLE
bug: `envd top` SIGSEGV

### ISSUE THREAD

**Issue by kemingy** [tensorchord/envd#725]
## Description

`evnd top`

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xeb0780]

goroutine 1 [running]:
github.com/tensorchord/envd/pkg/metrics.(*WidgetRow).GetRect(0xc00063f678?)
        <autogenerated>:1
github.com/gizak/termui/v3.Render({0xc00003a280?, 0x2, 0x1a53878?})
        github.com/gizak/termui/v3@v3.1.0/render.go:23 +0x8d
github.com/tensorchord/envd/pkg/app.top(0xc000659200)
        github.com/tensorchord/envd/pkg/app/top.go:83 +0x3e5
github.com/urfave/cli/v2.(*Command).Run(0x1a0a1a0, 0xc000658c00)
        github.com/urfave/cli/v2@v2.11.1/command.go:169 +0x631
github.com/urfave/cli/v2.(*App).RunContext(0xc000578c40, {0x12d4740?, 0xc00003e108}, {0xc00003a040, 0x2, 0x2})
        github.com/urfave/cli/v2@v2.11.1/app.go:378 +0xf15
github.com/urfave/cli/v2.(*App).Run(...)
        github.com/urfave/cli/v2@v2.11.1/app.go:251
main.run({0xc00003a040, 0x2, 0x2})
        github.com/tensorchord/envd/cmd/envd/main.go:38 +0x145
main.main()
        github.com/tensorchord/envd/cmd/envd/main.go:64 +0x2e
```

## Additional Info

```
envd: v0.2.0-alpha.11+5587c3f
  BuildDate: 2022-08-05T06:35:13Z
  GitCommit: 5587c3fa6f70e465df5ba14f3cec2f99339f7bcb
  GitTreeState: clean
  GoVersion: go1.18.4
  Compiler: gc
  Platform: linux/amd64
```

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by aseaday** [comment]
Sounds wired to me. I will fix it asap and add test.

**Comment by aseaday** [comment]
It has been fixed in #726

**Comment by gaocegege** [comment]
Never mind. Bugs are always here.
